.github/workflows/linting.yml

```diff
@@ -0,0 +1,21 @@
+name: Linting
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
We should probably run this only if there are changes in `doc` directory.

Reply:
Yes, that's on my list to do 👍